.travis.yml

```diff
@@ -0,0 +1,33 @@
+sudo: required
+
+language: scala
+
+services:
+- docker
+
+cache:
+  directories:
+  - $HOME/.ivy2/cache
+  - $HOME/.sbt/boot/
+
+script:
+- mkdir -p ${HOME}/.sbt
+- docker run -v ${HOME}/.ivy2:/root/.ivy2 -v ${HOME}/.sbt:/root/.sbt -v ${PWD}:/usace-flood-geoprocessing
```

Comment:
Caching stuff is to help make the builds a bit faster by downloading them all from S3 vs. from all of the sources `sbt` tries to wrangle together.

Also, here is an example of pinning the JVM and Scala versions. We should just stick to the combination we're deploying on (most likely OpenJDK 7 + Scala 2.10.5):

```yaml
jdk:
- openjdk7
- oraclejdk7
- oraclejdk8

scala:
- "2.10.5"
- "2.11.5"
```